test/integration/fixtures/parallel/testworkerid1.mjs

```diff
@@ -0,0 +1,5 @@
+describe('test1', () => {
+  it('should fail', () => {
+    throw new Error(`MOCHA_WORKER_ID=${process.env.MOCHA_WORKER_ID}`);
```

Comment:
You could test for `assert.ok(process.env.MOCHA_WORKER_ID === '0')`, etc.  and then in the spec test you have to check only for three passing tests. Would be much simpler.

Reply:
Ah! I was not sure it was ok to assert in the fixtures, I'll do that, thanks.

Reply:
done